apps/generic_terminal/unix_utilities.py

```diff
@@ -0,0 +1,90 @@
+from talon import Context, Module
```

Comment:
This file should probably go in `tags/terminal/` now since that refactoring a little while ago.

Reply:
Huh, `git rebase` was smart enough to do that automatically. Adjusted in the latest iteration.